### PR TITLE
add transactions command to siac

### PR DESCRIPTION
**Pull request by DavidVorick**

### CODE REVIEW

File: siac/walletcmd.go [hunk 1]
```diff
@@ -373,6 +381,61 @@ Siafund Claims:      %v SC
...
+// providing a net flow of siacoins and siafunds for each.
+func wallettransactionscmd() {
+	wtg := new(api.WalletTransactionsGET)
+	err := getAPI("/wallet/transactions?startheight=0&endheight=10000000", wtg)
+	if err != nil {
+		fmt.Println("Could not fetch transaction history:", err)
+		return
+	}
+
+	fmt.Println("[height]\t[transaction id]\t\t\t\t\t\t\t[net siacoin transfer]\t[net siafund transfer]")
```
Comment (lukechampine) [inline siac/walletcmd.go]: tabs aren't constant length; need to use spaces

File: siac/walletcmd.go [hunk 2]
```diff
@@ -373,6 +381,61 @@ Siafund Claims:      %v SC
...
+				incomingSiafunds = incomingSiafunds.Add(output.Value)
+			}
+		}
+
+		// Convert the siacoins to a float.
+		incomingSiacoinsFloat, _ := new(big.Rat).SetFrac(incomingSiacoins.Big(), types.SiacoinPrecision.Big()).Float64()
+		outgoingSiacoinsFloat, _ := new(big.Rat).SetFrac(outgoingSiacoins.Big(), types.SiacoinPrecision.Big()).Float64()
+
+		// Print the results.
+		fmt.Printf("%v\t\t%v\t %.2f SC\t\t", txn.ConfirmationHeight, txn.TransactionID, incomingSiacoinsFloat-outgoingSiacoinsFloat)
```
Comment (lukechampine) [inline siac/walletcmd.go]: see if you can use a width specifier to do the alignment, something like `"%10v %v %10.2f SC"`

File: siac/walletcmd.go (hunk 1 above)
Comment (DavidVorick) [inline siac/walletcmd.go]: maybe I should just not worry about formatting the first line until we can get a more consistent unit

File: siac/walletcmd.go (hunk 2 above)
Comment (DavidVorick) [inline siac/walletcmd.go]: actually, it looks like you can do that... nice!

**Comment by DavidVorick** [comment]
fixed up:  use spaces not tabs, and have everything align correctly always

**Pull request merged**